spec/support/coverage_helper.rb

```diff
@@ -0,0 +1,43 @@
+module Spec
+  module CoverageHelper
+    GEM_ROOT = Pathname.new(__dir__).join("../..").expand_path
+
+    class << self
+      attr_accessor :coverage_counter
+    end
+    self.coverage_counter = 0
+
+    def self.fix_coverage_resultset_paths
+      file = GEM_ROOT.join("coverage/.resultset.json")
+      contents = File.read(file)
+      contents.gsub!(%r{(?<=").+/\.bundle/plugin/bundler/gems/bundler-inject-\h+}, GEM_ROOT.to_s)
```

Comment:
What's happening here is that the coverage path ends up being the app's tmp dir + the .bundle/plugin clone dir.  In order to allow simplecov to merge the results properly, we have to change them so that they are relative to the gem root dir instead.

Reply:
Is there not a config option in `simplecov` to do this is part of the `coverage_prelude`?  So a `gsub` is not necessary?

I am fine if there isn't, just curious.

Reply:
Unfortunately no.  I tried searching all through that. 😦

Reply:
Alright 😞

Reply:
Wow, this one line feels like it consumed many brain cells and hours to figure out.  Maybe put your comment above in the code because I won't remember this when I look at this code again.